payments-core/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerInitializer.kt

```diff
@@ -67,12 +68,16 @@ internal class DefaultFlowControllerInitializer(
         }.fold(
             onSuccess = { stripeIntent ->
                 val paymentMethodTypes = stripeIntent.paymentMethodTypes.mapNotNull {
-                    PaymentMethod.Type.fromCode(it)
+                    SupportedSavedPaymentMethod.fromCode(it)
```

Comment:
Are tests needed here?

Reply:
Testing for the payment method types we're querying, and filtering out invalid payment methods.